addons/nuxeo-drive-server/nuxeo-drive-operations/src/test/java/org/nuxeo/drive/operations/TestIntegrationTestOperations.java

```diff
@@ -206,32 +184,22 @@ public void testIntegrationTestsSetupAndTearDown() throws IOException {
         // ----------------------------------------------------------------------
         String sarahCredentials = testUserCrendentialsArray[0];
         String sarahPassword = sarahCredentials.substring(sarahCredentials.indexOf(':') + 1);
-        Session unauthorizedSession = automationClient.getSession("drivesarah", sarahPassword);
-        try {
-            unauthorizedSession.newRequest(NuxeoDriveSetupIntegrationTests.ID)
-                               .set("userNames", "john,bob")
-                               .set("permission", "ReadWrite")
-                               .execute();
-            fail("NuxeoDrive.SetupIntegrationTests operation should not be callable by a non administrator.");
-        } catch (Exception e) {
-            // Expected
-        }
-
+        service = getAutomationServiceFor("drivesarah", sarahPassword);
+        getOperationError(NuxeoDriveSetupIntegrationTests.ID, Map.of( //
+                "userNames", "john,bob", //
+                "permission", "ReadWrite"), SC_NOT_FOUND);
         // ----------------------------------------------------------------------
         // Try to tear down the integration tests environment as an unauthorized
         // user => should fail
         // ----------------------------------------------------------------------
-        try {
-            unauthorizedSession.newRequest(NuxeoDriveTearDownIntegrationTests.ID).execute();
-            fail("NuxeoDrive.TearDownIntegrationTests operation should not be callable by a non administrator.");
-        } catch (Exception e) {
-            // Expected
-        }
+        getOperationError(NuxeoDriveTearDownIntegrationTests.ID, null, Map.of(), SC_NOT_FOUND);
```

Comment:
why not checking the assertion on error to fail the test like  : `assertEquals("NuxeoDrive.TearDownIntegrationTests operation should not be callable by a non administrator.", error);`  ?